Makefile

```
@@ -0,0 +1,115 @@
+	common/resample/src/kaiser50sinc.o \
+	common/resample/src/kaiser70sinc.o \
+	common/resample/src/makesinckernel.o \
+	common/resample/src/resamplerinfo.o \
+	common/resample/src/u48div.o \
+	common/rateest.o \
+	common/skipsched.o \
+	common/videolink/rgb32conv.o \
+	common/videolink/vfilterinfo.o \
+	common/videolink/vfilters/catrom2x.o \
+	common/videolink/vfilters/catrom3x.o \
+	common/videolink/vfilters/kreed2xsai.o \
+	common/videolink/vfilters/maxsthq2x.o \
+	common/videolink/vfilters/maxsthq3x.o
+
+LIB_OBJECTS = \
+	libgambatte/src/bitmap_font.o \
+	libgambatte/src/cpu.o \
+	libgambatte/src/gambatte.o \
+	libgambatte/src/initstate.o \
+	libgambatte/src/interrupter.o \
+	libgambatte/src/interruptrequester.o \
+	libgambatte/src/loadres.o \
+	libgambatte/src/memory.o \
+	libgambatte/src/sound.o \
+	libgambatte/src/state_osd_elements.o \
+	libgambatte/src/statesaver.o \
+	libgambatte/src/tima.o \
+	libgambatte/src/file/file_zip.o \
+	libgambatte/src/file/unzip/unzip.o \
+	libgambatte/src/file/unzip/ioapi.o \
+	libgambatte/src/mem/cartridge.o \
+	libgambatte/src/mem/memptrs.o \
+	libgambatte/src/mem/pakinfo.o \
+	libgambatte/src/mem/rtc.o \
+	libgambatte/src/sound/channel1.o \
+	libgambatte/src/sound/channel2.o \
+	libgambatte/src/sound/channel3.o \
+	libgambatte/src/sound/channel4.o \
+	libgambatte/src/sound/duty_unit.o \
+	libgambatte/src/sound/envelope_unit.o \
+	libgambatte/src/sound/length_counter.o \
+	libgambatte/src/video.o \
+	libgambatte/src/video/ly_counter.o \
+	libgambatte/src/video/lyc_irq.o \
+	libgambatte/src/video/next_m0_time.o \
+	libgambatte/src/video/ppu.o \
+	libgambatte/src/video/sprite_mapper.o
+
+TEST_OBJECTS = \
+	test/testrunner.o
+
+all: $(SDL)
+
+SDL_LFLAGS != $(PKG_CONFIG) --libs sdl
+$(SDL): $(SDL_OBJECTS) $(LIB)
+	$(CXX) $(CXXFLAGS) -o $@ -pthread $(SDL_OBJECTS) $(LIB) -lz \
+		$(SDL_LFLAGS) $(LDFLAGS)
+
+$(LIB): $(LIB_OBJECTS)
+	$(AR) $(ARFLAGS) $@ $(LIB_OBJECTS)
+	$(RANLIB) $@
+
+PKG_CONFIG = pkg-config
```

declared too late... it will make line 74 fail silently